src/assets/scripts/modules/download-filter/download-filter.js

```diff
@@ -0,0 +1,59 @@
+import List from 'list.js';
+
+const component = document.querySelector('[data-downloads]');
+const filter = document.querySelector('[data-downloads-filter]');
+const sort = document.querySelector('[data-downloads-sort]');
+const formatButtons = Array.from(document.querySelectorAll('[data-downloads-format]'));
+
+function initDownloadFilter(formats) {
+  if(!component || !filter || !sort || !formats || !formatButtons) {
+    return
+  }
+
+  const options = {
+    listClass: 'js-downloads-list',
+    valueNames: [
+      { data: ['name'] },
+      { data: ['format'] }
+    ]
+  };
+
+  const list = new List(component, options);
+  let sortActive
+  let activeFormats = []
+
+  sort.addEventListener('click', handleSort);
+  filter.addEventListener('submit', event => event.preventDefault());
+  formatButtons.forEach(format => format.addEventListener('click', () => handleFormat(format) ))
```

Comment:
if you loop over formatButtons, should the `format` item not be `formatButton`?

Reply:
refactored to use `formatButton`